doc/frequent-itemsets.rst

```diff
@@ -0,0 +1,69 @@
+Frequent Itemsets
+=================
+
+.. figure:: icons/frequent-itemsets.png
+
+Finds frequent itemsets in the data.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Data**
+
+Data set
+
+
+**Outputs**:
+
+-  **Matching Data**
+
+Data instances matching the criteria.
+
+
+Description
+-----------
+
+The widget finds frequent items in a data set based on a measure of
+support for the rule.
+
+.. figure:: images/frequent-itemsets-stamped.png
+
+1. Information on the data set. 'Expand all' expands the frequent itemsets
+tree, while 'Collapse all' collapses it.
+
+2. In *Find itemsets by* you can set criteria for itemset search:
+
+   -  **Minimal support**: support is a ratio between how often a rule is true and
+      the entire number of examples in a data set. For large data sets it is 
+      normal to set a lower minimal support (e.g. between 2%-0.01%).
+   -  **Max. number of itemsets**: limits the upward quantity of itemsets. Minimum value
+      is 10000; which means only the 10000 top-performing itemsets will be displayed.
```

Comment:
Avoid specifics, those are most prone to change.

... limits the upward quntity of generated itemsets. Itemsets are generated in no particular order.